### DISCUSSION TITLE
Evaluating FEX for Northstar

### DISCUSSION THREAD

**Discussion by Jan200101** [R2Northstar/Northstar#590] [Research]
Previously there were some tests done using box64 but there is an alternative out there called FEX which, accordingly to some people, is more performant than box64

Discord forum post on the same topic: https://discord.com/channels/920776187884732556/1175007936184070206/1175007936184070206

**Comment by GeckoEidechse** [comment]
Link to project https://github.com/FEX-Emu/FEX
